README.md

```diff
@@ -1,130 +1,211 @@
 # auto-minify
-Minifies JS and CSS files with Babel-Minify and CleanCSS
 
+![test workflow](https://github.com/nizarmah/auto-minify/actions/workflows/test.yml/badge.svg?branch=master&event=push)
+
+Minifies JS and CSS files.
+
+## Quickstart
+
+```yaml
+on: [push]
+
+jobs:
+  minify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check out the repository
+        uses: actions/checkout@v4
+
+      - name: auto-minify the files
+        uses: nizarmah/auto-minify@v3
+
+      - name: auto commit the minified files
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: "bot: Add auto minified files"
+          branch: ${{ github.ref }}
```

Comment:
The "branch" isn't needed.

Based on the main repo, it defaults to the "current" branch.

Reply:
ac8058cac187ec84051a7b2081922e85406fe6d0